engine/movie/hall_of_fame.asm

```diff
@@ -200,7 +200,7 @@ HoFLoadPlayerPics:
 	ld de, RedPicFront
 	ld a, BANK(RedPicFront)
 	call UncompressSpriteFromDE
-	ld a, $0
+	ld a, SRAM_DISABLE
 	call OpenSRAM
```

Comment:
`OpenSRAM:` takes a bank, not an enable/disable flag. Although this change came from a suggestion on your first commit.. in general it doesn't look like your intention was to "fix" all the calls to `OpenSRAM` in this PR, so you could just revert this if you don't want to fix all of them.